arduino/TFT_Matrix/TFT_Matrix.ino

```diff
@@ -19,12 +22,12 @@
 TFT_eSPI tft = TFT_eSPI();       // Invoke custom library
 
 #define TEXT_HEIGHT 8 // Height of text to be printed and scrolled
-#define BOT_FIXED_AREA 0  // Number of lines in bottom fixed area (lines counted from bottom of screen)
-#define TOP_FIXED_AREA 0  // Number of lines in top fixed area (lines counted from top of screen)
+#define BOT_FIXED_AREA 40  // Number of lines in bottom fixed area (lines counted from bottom of screen)
+#define TOP_FIXED_AREA 40  // Number of lines in top fixed area (lines counted from top of screen)
 
 uint16_t yStart = TOP_FIXED_AREA;
-uint16_t yArea = 240 - TOP_FIXED_AREA - BOT_FIXED_AREA;
-uint16_t yDraw = 240 - BOT_FIXED_AREA - TEXT_HEIGHT;
+uint16_t yArea = 320 - TOP_FIXED_AREA - BOT_FIXED_AREA;
```

Comment:
Height must stay unchanged for ST7789 for scrolling to work. But the display does not have this resolution, therefore add BFA and TFA.